src/app/models/weather-alert.model.ts

```diff
@@ -125,6 +128,37 @@ export class WeatherAlertResponse {
       return returnedDescription;
     }
   }
+
+  /**
+   * @description - passes weather alert and loops through the event value to match pre determined key words that will be used to grab info from a local json
+   * @param {WeatherAlertInfo} weatherAlert - active passed weather alert
+   * @returns {SafetyInfo} - returns an object of weather related safety info
+   */
+  public doesWeatherAlertRelateToSafetyJobAid(weatherAlert: WeatherAlertInfo): SafetyInfo {
+    let tempTag: string;
+    const coldWeatherTags: string[] = ['Ice', 'Cold', 'Snow', 'Winter', 'Freez', 'Blizzard', 'Frost', 'Wind Chill']; // spelling of 'freez' is intentional as the event name from api may say 'freeze' or 'freezing'
```

Comment:
I thought about trying to do these with regex. Would only be worth it if it showed significant efficiency improvement. Did some research. It is really hard to say which method is best, I say we just leave it. However, according to this, `.indexOf(str) === -1` is extremely fast in chrome!
https://stackoverflow.com/questions/5296268/fastest-way-to-check-a-string-contain-another-substring-in-javascript

Reply:
Let's tak abou these